### ISSUE TITLE
BUG edit in place

### ISSUE THREAD

**Issue by julienpillias** [php-translation/symfony-bundle#141]
Hi,

I'm having troubles with the edit in place, wich works well in some pages, and not on others.

For example, i'm having this JS error on one page : 
 ```
content-tools.min.js:3 Uncaught DOMException: Failed to execute 'setAttribute' on 'Element': 'light-blue"' is not a valid attribute name.
    at c.a.Text.c.mount (http://mywebsite.fr/bundles/translation/js/content-tools.min.js:3:20706)
    at new c (http://mywebsite.fr/bundles/translation/js/content-tools.min.js:3:12341)
    at c._initRegions (http://mywebsite.fr/bundles/translation/js/content-tools.min.js:6:24008)
    at c.start (http://mywebsite.fr/bundles/translation/js/content-tools.min.js:6:20300)
    at b.<anonymous> (http://mywebsite.fr/bundles/translation/js/content-tools.min.js:6:14355)
    at b.ComponentUI.a.dispatchEvent (http://mywebsite.fr/bundles/translation/js/content-tools.min.js:4:24074)
    at b.IgnitionUI.b.edit (http://mywebsite.fr/bundles/translation/js/content-tools.min.js:4:27817)
    at HTMLDivElement.<anonymous> (http://mywebsite.fr/bundles/translation/js/content-tools.min.js:4:29468)
```

Searching in code source for light-blue give's me some span, like : 
```
<span class="light-blue">Sorted.</span>
```

Any idea ? 

Thank you for your help

**Comment by julienpillias** [comment]
Hi,

Just to help about debug : when activiing edit in place, some title like : 

```
<h1>La maison que vous<br> méritez. <span class="light-blue">Sorted.</span></h1>
```
http://hpics.li/14fb6f6


becomes : 
```
<h1><x-trans data-key="pages|index.main_title" data-value="La maison que vous<br> méritez. <span class=" light-blue"="">Sorted." data-plain="La maison que vous<br> méritez. <span class="light-blue">Sorted.</span>" data-domain="pages" data-locale="fr"&gt;La maison que vous<br> méritez. <span class="light-blue">Sorted.</span></x-trans></h1>
```
and breaks the page
http://hpics.li/efc1f67

**Comment by rvanlaak** [comment]
Ah so it seems you use html in your translations? That probably is what's going on, putting html in there makes it hard for the `data-value` attribute. @julienpillias does your page actually still looks okay? Can imagine that's broken too?

**Comment by julienpillias** [comment]
Exact, some part of the page is a bit broken ! 
So edit in place is not able to handle HTML inside a "simple" block ? 

Just tried with translate name ".html",  no more JS erros, but not able to have HTML like : 
```
La maison que vous<br> méritez. <span class="light-blue">Sorted.</span>
``` 
and edit in place still bugging, even if I wrap it with a <p>

**Comment by damienalexandre** [comment]
Hi,
we are going to have difficulties making Edit In Place works with HTML translations, 
but as it a WYSIWYG editor if should be possible.

Also, the response listener is supposed to remove malformed HTML:

https://github.com/php-translation/symfony-bundle/blob/aae58657e6239c935db803e32a73df609020225b/EventListener/EditInPlaceResponseListener.php#L88-L101

Sorry for your troubles, I suggest you split your translation in multiple keys for now.

**Comment by julienpillias** [comment]
Hi,

Setting show_untranslatable to false does not resolve the problem.
I'll try to split my translations.

Since edit in place does not work so well, do you think it's possible to have a WYSIWYG in webUI and/or profiler ?

**Comment by odolbeau** [comment]
Looks like it has been fixed by #259
